scripts/demoScripts/reportsMockScript.js

```diff
@@ -0,0 +1,175 @@
+const Client = require('node-rest-client').Client;
+const { createMoodEntries, createMeditationEntries, createMedicationAdherence } =  require('./createRecordsScript');
+
+//URL's
+const authUrl = `${process.argv[2]}/auth/login`;  //"http://localhost:4000/api/v1/auth/login"
```

Comment:
Low prio: Assuming you're doing this to get the code out the door quick. Fine.

FFR:
Positional argv for this many args is inappropriate (esp. without a `--help` that shows the argv pos). Positional argv `--authUrl <>` would be more appropriate.

Don't bother with argv parsing; use a light-weight library like minimalist or yargs.